WcaOnRails/app/models/user.rb

```diff
@@ -1251,4 +1251,8 @@ def senior_or_self
   def is_delegate_in_probation
     Role.where(user_id: self.id).where("end_date is null or end_date >= curdate()").present?
   end
+
+  def can_view_wfc_matters?
```

Comment:
There is already `user.can_admin_finances?` which I believe can be reused for this PR